### PR TITLE
[Rest Server] Add --shm-size in docker run

### PR DESCRIPTION
**Pull request by abuccts**
Add `--shm-size` in docker run. Set shared memory size the same as cpu memory.

### CODE REVIEW

File: rest-server/src/templates/yarnContainerScript.mustache [hunk 1]
```diff
@@ -208,6 +208,7 @@ docker run --name $docker_name \
   --network=host \
   --cpus={{{ taskData.cpuNumber }}} \
   --memory={{{ taskData.memoryMB }}}m \
+  --shm-size={{{ taskData.memoryMB }}}m \
   $nvidia_devices \
```
Comment (sterowang) [inline rest-server/src/templates/yarnContainerScript.mustache]: What's specific problem need to solve to add this switch?

Comment (abuccts) [reply rest-server/src/templates/yarnContainerScript.mustache]: pytorch needs to use shm when reading data

**Comment by abuccts** [comment]
It's ok to expose this field in json file, but we'd better check the value, e.g. `shmsize <= memory * 0.5`?

**Comment by fanyangCS** [comment]
> # PAI 深度学习任务指南

this file will be removed. pls exclude it in this PR.

---
Refers to: job-tutorial/README-zh.md:19 in f9ea61b. [](commit_id = f9ea61b1678a76fadf2562ec173c9cfb2963719c, deletion_comment = False)

File: job-tutorial/README.md [hunk 2]
```diff
@@ -112,6 +113,7 @@ Below please find the detailed explanation for each of the parameters in the con
 | `taskRole.taskNumber`          | Integer, required          | Number of tasks for the task role, no less than 1 |
 | `taskRole.cpuNumber`           | Integer, required          | CPU number for one task in the task role, no less than 1 |
 | `taskRole.memoryMB`            | Integer, required          | Memory for one task in the task role, no less than 100 |
+| `taskRole.shmMB`               | Integer, optional          | Shared memory for one task in the task role, no more than memory size |
```
Comment (fanyangCS) [inline job-tutorial/README.md]: >Shared [](start = 64, length = 6)

please state the default value here.

Comment (abuccts) [reply job-tutorial/README.md]: updated

**Pull request merged**